src/middlewares/userAcess.js

```diff
@@ -0,0 +1,17 @@
+import services from '../services';
+import helpers from '../helpers';
+
+const { userServices: { findUserRole } } = services;
+const { errorResponse } = helpers;
+
+const verifySuperAdmin = async (req, res, next) => {
```

Comment:
i think we already have a middleware called authorizeUser that can do this same function

Reply:
Yeah, will rebase